src/task/task.json

```diff
@@ -55,6 +55,15 @@
             },
             "groupName": "configurationOptions"
         },
+        {
+            "name": "FailBuildOnCodeIssues",
+            "type": "boolean",
+            "label": "Fail build when Code Issues are found",
+            "defaultValue": "true",
+            "required": false,
+            "helpMarkDown": "If selected, the build fails if there are any Code Issue equal or above the Fail build severity.",
```

Comment:
Should be `If selected, the build fails if there are any Code Issue(s) equal or above the Fail build severity.`

Reply:
Well spotted. But I think that `any Code Issues` is the grammatically correct version.